tests/unit/controllers/projects/edit/steps-test.js

```diff
@@ -0,0 +1,12 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Controller | projects/edit/steps', function(hooks) {
+  setupTest(hooks);
+
+  // Replace this with your real tests.
+  test('it exists', function(assert) {
+    const controller = this.owner.lookup('controller:projects/edit/steps');
+    assert.ok(controller);
+  });
```

Comment:
@andycochran looks great! You can write a test here. If you need to, you can stub in that router service:

```javascript
this.owner.register('service:router', {
  currentRouteName: 'index'
});

const routerServiceStub = this.owner.lookup('service:router');

routerServiceStub.set('currentRouteName', 'projects.edit.steps.rezoning');

assert.equal(controller.get('currentStepNumber'), 3);
```